TLM/TMPE.API/Traffic/Enums/TrafficLightType.cs

```diff
@@ -0,0 +1,9 @@
+namespace TrafficManager.API.Traffic.Enums {
+    public enum TrafficLightType {
```

Comment:
I don't think `TrafficLightType` is the right name for this. It includes things like "paused."

It's pretty UI-centric. Maybe its name should reflect that?

Reply:
what should I call it?

Reply:
`TrafficLightFlags`?

Reply:
Paused traffic light acts like normal vanilla traffic light.
I also have the option of dropping paused.

Reply:
And no, I say don't drop paused. Under my interpretation, it belongs there.